test/NuGet.Tests.Apex/NuGet.Tests.Apex/Utility/CommonUtility.cs

```diff
@@ -400,7 +400,9 @@ private static LockFile GetAssetsFileWithRetry(string path)
                     {
                         content = File.ReadAllText(path);
                         var format = new LockFileFormat();
+#pragma warning disable CS0612 // Type or member is obsolete
                         return format.Parse(content, path);
+#pragma warning restore CS0612 // Type or member is obsolete
```

Comment:
this code looks easy to convert to a stream based overload, so it doesn't need to use an obsolete overload. Might make our tests marginally faster too, not needing to load the entire file into a LOH string.